src/Microsoft.DocAsCode.MarkdigEngine.Extensions/TripleColon/TripleColonRenderer.cs

```diff
@@ -9,6 +9,11 @@ public class TripleColonRenderer : HtmlObjectRenderer<TripleColonBlock>
     {
         protected override void Write(HtmlRenderer renderer, TripleColonBlock b)
         {
+			if (b.Extension.Render(renderer, b))
+			{
+				return;
+			}
+
```

Comment:
can you restore the 2 space indents in all the modified files? We'll have strange diffs going forward if we don't keep the whitespace rules consistent.

Reply:
Yeah, I keep forgetting this.